xarray_beam/_src/core.py

```diff
@@ -269,13 +282,14 @@ def __init__(
     """Initialize DatasetToChunks.
 
     Args:
-      dataset: dataset to split into (Key, xarray.Dataset) pairs.
+      dataset: dataset or datasets to split into (Key, xarray.Dataset) or
+        (Key, (xarray.Dataset, ...)) pairs.
```

Comment:
I might consider using `List[xarray.Dataset]` rather than `Tuple[xarray.Dataset, ...]`.

Lists are the more idiomatic type for homogeneous sequences of arbitrary length in Python, e.g., as evidenced by how the syntax for list comprehensions is more succinct.